webpack.config.js

```diff
@@ -0,0 +1,126 @@
+const HtmlWebPackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const path = require('path');
+const TerserJSPlugin = require('terser-webpack-plugin');
+const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
+
+module.exports = {
+  entry: './index.jsx',
+  output: {
+    path: path.join(__dirname, '/dist'),
+    filename: 'index.js'
+  },
+  context: path.join(__dirname, 'src'),
+  resolve: {
+    extensions: ['.js', '.ts', '.tsx', '.css', '.jsx', '.scss'],
```

Comment:
Usually css and another style extensions are not including in resolve extensions because it makes your code confusing (think why ;)